test/node/search.js

```diff
@@ -61,9 +65,28 @@ describe('Search Tests', () => {
         assert(searchId2);
     });
 
-    it('should confirm search results for the user', async () => {
-        //wait to allow time for search to return
-        await helper.sleep(6000);
+    it('should search for user using phone number and raise searchStatus event', async () => {
```

Comment:
Should the phone number be set for the user in the preparation too? It might not be enough time to index the phone number for it to be searchable.

Reply:
I didn't run into that problem but I can do that.